src/bindings/python/CMakeLists.txt

```diff
@@ -271,6 +271,7 @@ macro(ov_define_setup_py_dependencies)
     endforeach()
 
     file(GLOB_RECURSE openvino_py_files ${OpenVINOPython_SOURCE_DIR}/src/openvino/*.py)
+    file(GLOB_RECURSE openvino_py_files ${OpenVINOPython_SOURCE_DIR}/src/openvino/*.pyi)
```

Comment:
here you override previously set `openvino_py_files`

Reply:
Fixed, thanks!